src/5.0_to_5.1/migrateFilters.test.ts

```diff
@@ -1,12 +1,15 @@
 import { parse, stringify } from "@activeviam/activeui-sdk-5.0";
+import { sandboxDataModel } from "@activeviam/data-model-5.1/dist/__test_resources__";
 import { migrateFilters } from "./migrateFilters";
 
+const dataModels = { sandbox: sandboxDataModel };
+
 describe("migrateFilters", () => {
   it("wraps the MDX representing each filter", () => {
```

Comment:
Would be cool to add two other tests:
- one with `cubeName` explicitly passed to `migrateFilters`, to cover that case
- one where no cube is matching the `mdx`, to illustrate the logic of `createFilterWithFirstMatchingCube` and in which case it would throw